frontend/ui/translator.lua

```diff
@@ -146,6 +146,7 @@ local Translator = {
     trans_servers = {
         "https://translate.googleapis.com/",
         -- "http://translate.google.cn",
+    "https://fanyi.youdao.com",
```

Comment:
4 spaces for indentation please, not 2. Besides that I assume it's mostly just copied from the other one?